stock_split_picking/models/stock_picking.py

```diff
@@ -78,3 +80,19 @@ def split_process(self):
                     'picking_id': backorder_picking.id,
                 })
                 new_moves._action_assign()
+
+                # Propagate split to chained picking/moves
+                chained_moves = picking.move_lines.mapped('move_dest_ids')
+                for chained_move in chained_moves:
+                    # import pdb; pdb.set_trace()
+                    chained_picking = chained_move.picking_id
+                    chained_picking.action_assign()
+                    for move_line in chained_move.move_line_ids:
+                        # FIXME is it the right way to propagate done qty to
+                        # the chained moves?
+                        qty_done = product_qty_done[move_line.product_id.id]
+                        move_line.write({
+                            'qty_done': qty_done,
+                            'product_uom_qty': qty_done,
+                        })
+                    chained_picking.split_process()
```

Comment:
Before calling this, you probably have to write on chained_move.qty_done, otherwise it will raise the error on line 23.